.rubocop.yml

```diff
@@ -4,5 +4,8 @@ inherit_gem:
 Naming/AccessorMethodName:
   Enabled: false
 
+Style/ExplicitBlockArgument:
+  Enabled: false
+
```

Comment:
I think there may be a bug here. This rule was changing

```ruby
@config_lambda = ->(connection) { yield(connection) }
```

to

```ruby
@config_lambda = ->(&block)
```

which are not at all the same